### PR TITLE
Add support for .NET Framework 4.7.2 and .NET Framework 4.8 to NetFxExten…

### PR DESCRIPTION
**Pull request by FroggieFrog**
…sion.
Based on https://github.com/wixtoolset/issues/issues/5575#issuecomment-386223409 and https://github.com/wixtoolset/issues/issues/5977#issuecomment-485747260

### CODE REVIEW

File: src/ext/NetFxExtension/wixlib/NetFxExtension.wixproj [hunk 1]
```diff
@@ -25,6 +25,8 @@
     <Compile Include="NetFx46.wxs" />
     <Compile Include="NetFx461.wxs" />
     <Compile Include="NetFx462.wxs" />
+	<Compile Include="NetFx472.wxs" />
```
Comment (TheStormN) [inline src/ext/NetFxExtension/wixlib/NetFxExtension.wixproj]: The indentation is different. :)

Comment (FroggieFrog) [reply src/ext/NetFxExtension/wixlib/NetFxExtension.wixproj]: should be resolved

**Comment by HelioVaGator** [comment]
Hi, when are we planning to merge this and release it?

**Comment by rseanhall** [comment]
Thanks, but the hashes have changed. I sent a new pull request with up to date hashes at https://github.com/wixtoolset/wix3/pull/520.

**Pull request closed without merging**